packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -50,6 +61,9 @@ const FormLabel = ({
   children,
   id,
   htmlFor,
+  contrast = 'low',
```

Comment:
We need the label to also follow contrast of the component where it's being used